core/templates/dev/head/pages/exploration_player/exploration_footer_directive.html

```diff
@@ -23,6 +23,37 @@
       padding-left: 0;
     }
 
+    .oppia-exploration-footer .mobile-feedback {
+      color: #fff;
+      list-style-type: none;
+      padding-left: 0;
+    }
+
+    .oppia-exploration-footer .mobile-feedback li > a {
+      margin-top: 2px;
+    }
+
+    .oppia-exploration-footer .mobile-feedback > .popover {
+      /* Inline styles are applied by bootstrap-ui javascript, since they have
+      the highest specificity only !important can override them. This can be
+      resolved by updating bootstrap-ui.
+       */
+      color: rgba(0,0,0,0.87);
+      margin-left: 5px;
+      margin-right: 5px;
+      max-width: 100vw;
+      right: 15px;
+      width: auto !important;
```

Comment:
> Inline styles are applied by bootstrap-ui javascript, since they have the highest specificity only !important can override them. This can be resolved by updating bootstrap-ui.

Agree, but as in here we are using `max-width` here we can format the width with specific selectors. This code  (without !important ) below should work fine. Please, confirm. 

```
    color: rgba(0,0,0,0.87);
    margin-left: 5px;
    margin-right: 5px;
    max-width: 97.6vw;
    right: 15px;
```